recipes/foursquare/init.sql

```diff
@@ -90,9 +90,9 @@ BEGIN
                     END)
                 from city_zip_boro a where zip=a.zipcode) as borocode,
                 categoryname as category,
-                ROUND(avg(CASE WHEN demo='All' THEN visits END),0) AS visits_avg_all,
-                ROUND(avg(CASE WHEN demo='Below65' THEN visits END),0) AS visits_avg_u65,
-                ROUND(avg(CASE WHEN demo='Above65' THEN visits END),0) AS visits_avg_o65
+                SUM(CASE WHEN demo='All' THEN visits END) AS visits_all,
```

Comment:
@AmandaDoyle  
this is a change of methodology, do we confirm that we are no longer taking average but the sum ?

Reply:
@SPTKL this is the daily data.  We shouldn't be taking the average number of visits.  Averages are only taken for weekly data.  
Let me know if I'm misunderstanding something and we can hop on a call